joomla/joomla-cms
# [5.1] Convert mod_tags_similar to service provider

### Summary of Changes

Converts the tags similar site module to service provider.


### Testing Instructions

1. Make sure you have site module mod_tags_similar published
2. Add some tags and assign them to several content items
3. You should see module output on content items, which have at least 1 tag assigned and this tag is assigned also on another content items.

![mod_tags_similar](https://github.com/joomla/joomla-cms/assets/59166858/67335bd7-6e88-4465-b439-1f0d9aa4a8f2)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
